app/controllers/admins/employees_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Admins
+  class EmployeesController < AdminController
+    def index
+      @employees = Employee.all
+    end
+
+    def edit
+      @employee = Employee.find(params[:id])
+      if admin_signed_in?
+        render 'admins/employees/edit'
+      else
+        flash[:alert] = 'You need to be loged as admin to perform this operation'
+        redirect_to admins_employees_path
+      end
+    end
+
+    def destroy
+      @employee = Employee.find(params[:id])
+      flash[:notice] = if @employee.destroy
+                         'Employee account was deleted successfully'
+                       else
+                         'Employee account deletion failed'
+                       end
+      redirect_to admins_employees_path
+    end
+
+    def update
+      @employee = Employee.find(params[:id])
+      if @employee.update(params.require(:employee).permit(:email, :password, :number_of_available_kudos))
```

Comment:
1. because when i leave it as it previously were `if @employee.update(employee_params)` I get the `undefined local variable or method `employee_params' for` error i know this isn't the perfect solution to the problem but you know what they say: "when something is stupid but it works --> it isn't stupid" :D

Reply:
you have an error in this file, that's why you cannot use the method `employee_params` inside of class `EmployeesController` scope. Look closely